cypress/integration/lpa/cases/administration.spec.js

```diff
@@ -0,0 +1,53 @@
+describe("LPA administration changes", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.loginAs("LPA Manager");
+    cy.createDonor().then(({ id: donorId }) => {
+      cy.createLpa(donorId).then(({ id: lpaId }) => {
+        cy.wrap(donorId).as("donorId");
+        cy.wrap(lpaId).as("lpaId");
+      });
+    });
+  });
+
+  it("should change the LPA receipt date", function () {
+    cy.visit(`/lpa/#/person/${this.donorId}/${this.lpaId}`);
+
+    cy.intercept({ method: "GET", url: "/api/v1/persons/*/events*" }).as(
+      "eventsRequest"
+    );
+
+    cy.intercept({ method: "PUT", url: "/api/v1/lpas/*" }).as(
+      "putRequest"
+    );
+
+    cy.wait("@eventsRequest");
+
+    cy.get("uib-tab-heading[id=Administration]").contains("Administration").click();
+    cy.contains("Edit Dates").click();
+
+    cy.wait(500);
```

Comment:
This is not ideal however without this, the date does not get changed which suggests that the field disabled for a brief moment as the form loads. `cy.get("#receiptDate0:not([disabled])")` would not work

Reply:
Oof yeah, seems like some pain introduced from the `opgDatepicker` directive. Not ideal, but that's fairly normal for these tests now 😞

Reply:
It looks like that page loads `/api/v1/cases/:id`, so you could might be able to intercept and wait for that request